headers/modsecurity/transaction.h

```diff
@@ -219,6 +221,8 @@ class TransactionAnchoredVariables {
     AnchoredVariable m_variableInboundDataError;
     AnchoredVariable m_variableMatchedVar;
     AnchoredVariable m_variableMatchedVarName;
+    AnchoredVariable m_variableMscPcreErrored;
```

Comment:
Given the name of MscPcreLimitsExceeded, I can understand the temptation to use the verb-past-tense here.

But for consistency with variables like MULTIPART_STRICT_ERROR, INBOUND_DATA_ERROR (etc.),